# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1228717858 -3600
#      Mon Dec 08 07:30:58 2008 +0100
# Branch release-3-0-x
# Node ID 4ccdb0d32158b0458fa98c8ac3d021f935b95bf8
# Parent  4364e3fab451aeb5320e38eff248569a63ce9ec2
fix runaway changelog entry

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -3,16 +3,21 @@ 2008-12-08  Jaroslav Hajek  <highegg@gma
 	* pt-select.cc (equal): Make empty matrices match.
 
 2008-11-12  David Bateman  <dbateman@free.fr>
 
 	* OPERATORS/op-cell.cc (op_catop_matrix_cell): Cast args in the correct
 	order.
 	* OPERATORS/op-struct.cc (op_catop_matrix_struct): Ditto.
 
+2008-10-31  David Bateman  <dbateman@free.fr>
+
+	* ov-base-scalar.h (octave_value octave_base_scalar::reshape 
+	(const dim_vector&) const): New method.
+
 2008-10-30  John W. Eaton  <jwe@octave.org>
 
 	* oct-map.cc (Octave_map::index): Copy key_list.
 
 2008-10-17  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/__qp__.cc (qp): Undo part of change from
 	2007-09-04 (compute Y at each iteration).
@@ -9353,21 +9358,16 @@ 2004-07-23  David Bateman  <dbateman@fre
 	(concat): New function.
 
 	* Cell.cc, Cell.h (insert): New function
 
 	* ov-base.cc, ov-base.h, ov-scalar.h, ov-complex.h, ov-bool.h, 
 	ov-range.h, ov-base-mat.h, (resize): New function.
 
 
-2008-10-31  David Bateman  <dbateman@free.fr>
-
-	* ov-base-scalar.h (octave_value octave_base_scalar::reshape 
-	(const dim_vector&) const): New method.
-
 2004-07-23  John W. Eaton  <jwe@octave.org>
 
 	* symtab.cc (whos_parameter): Move decl here, from symtab.h.
 
 2004-07-23  Oyvind Kristiansen  <oyvinkri@stud.ntnu.no>
 
 	* symtab.cc (symbol_record::subsymbols_list): New method.
 	(symbol_record::maybe_list): Call to subsymbols_list and merge
